.travis.yml

```diff
@@ -1,6 +1,24 @@
+sudo: false
+
 language: groovy
+
 jdk:
-- oraclejdk8
+  - oraclejdk7
+  - oraclejdk8
+
+env:
+  - GRADLE_VERSION=1.12
+  - GRADLE_VERSION=2.4
+  - GRADLE_VERSION=2.5
+
+before_script:
+  - cp -rf gradle/wrapper/gradle-wrapper.template gradle/wrapper/gradle-wrapper.properties
```

Comment:
I wonder wouldn't be enough to just append a line with `gradle-2.x-bin.zip` to that file (instead of using a template file)? As this is a properties file the last occurrence should be taken into account (it works fine with `gradle.properties` in micro-infra-spring acceptance tests with boot-microservice).